test/jdk/javax/swing/JComboBox/TestComboBoxHeight.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @key headful
+ * @bug 4517214
+ * @summary Tests that comboBox is not doubleheight if editable and has TitledBorder
+*/
+
+import javax.swing.BorderFactory;
+import javax.swing.JComboBox;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.SwingUtilities;
+import javax.swing.WindowConstants;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.Robot;
+
+public class TestComboBoxHeight {
+    private static String[] data = { "Ten", "Twenty", "Forty-three" };
+    private static JFrame jframe;
+    private static int heightCombo1, heightCombo2;
+    private static JComboBox combo1, combo2;
+
+    public static void main(String[] args) throws Exception {
+        try {
```

Comment:
Can we check all L&Fs?

Reply:
Not now..we already have open JBS for window L&F JDK-8233469 and it also fails in Nimbus which will be looked into separately..
I will like this PR to be for for default L&F only otherwise it will cause failure in CI systems..